### ISSUE TITLE
question: is a update for 4 planned?

### ISSUE THREAD

**Issue by dmaz** [samdze/godot-native-bullets-plugin#4]
Wow... This is super nice, great job!
I just started moving my projects over to 4 and was wondering if you had any plans in the near future to add a v4 branch for this?

**Comment by samdze** [comment]
Yes, I think I'll work on aa 4.0 version when the new GDExtension system will be stable enough.
I'm renaming this issue to track the development of the 4.0 version.

**Comment by Malkverbena** [comment]
Nice job dude!
Would be amazing a version for Godot 4!!

**Comment by samdze** [comment]
Yes I'm considering doing the porting when I have time!
